pkg/server/plugin/keymanager/v0.go

```diff
@@ -92,6 +97,28 @@ func (v0 *V0) makeKey(id string, pb *keymanagerv0.PublicKey) (Key, error) {
 	}, nil
 }
 
+func (v0 *V0) convertKeyType(t KeyType) (keymanagerv0.KeyType, error) {
+	switch t {
+	case KeyTypeUnset:
+		return keymanagerv0.KeyType_UNSPECIFIED_KEY_TYPE, v0.Error(codes.InvalidArgument, "key type is required")
+	case ECP256:
+		return keymanagerv0.KeyType_EC_P256, nil
+	case ECP384:
+		return keymanagerv0.KeyType_EC_P384, nil
+	case RSA2048:
+		return keymanagerv0.KeyType_RSA_2048, nil
+	case RSA4096:
+		return keymanagerv0.KeyType_RSA_4096, nil
+	default:
+		return keymanagerv0.KeyType_UNSPECIFIED_KEY_TYPE, v0.Errorf(codes.Internal, "facade does not support key type %s", t)
```

Comment:
nit: we use %q for this in other places, which I prefer
```suggestion
		return keymanagerv0.KeyType_UNSPECIFIED_KEY_TYPE, v0.Errorf(codes.Internal, "facade does not support key type %q", t)
```